lib/include/cpp-json/functional/cxx_check_prerequisites.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
I know it's ubiquitous, but I do tend to avoid `#pragma once` since it's non-standard.

Reply:
OK, I find this quite stringent, as all compilers support it. However, I respect your decision.